### DISCUSSION TITLE
Backup wallet interface.

### DISCUSSION THREAD

**Discussion by davelincoder** [FuelLabs/fuel-core#1681] [General]
I know fuel is considering security in its ecosystem and I love the idea that the team considered and worked toward a fuel wallet. I would suggest you this process. The process is when a newly created wallet within the same wallet can continuously read transactions from the old main wallet. This idea is just how Galxe allows you to add a secondary wallet to your already existing primary wallet. I was hacked recently, and I must confess it has been difficult correlating my on-chain activities from my old wallet to my new wallet because most projects don't have the tech for this.

**Comment by xgreenx** [comment]
I'm closing this discussion since I don't think that it is the right repository for this discussion. Maybe you want to consider more proper place like: https://github.com/FuelLabs/fuels-wallet
